www/index.md

```diff
@@ -24,6 +24,11 @@ Nächstes Treffen
 
 {% assign termin = page.termine | first %}
 
+Neugierig?
+===
+
+Komm einfach vorbei, der Treff ist offen für alle, neue Gesichter sind immer gerne gesehen!
```

Comment:
also ich würde da ja statt dem ! ein :) machen :)

Reply:
oder ein `.`

Reply:
@koebi so?
`Komm einfach vorbei, der Treff ist offen für alle, neue Gesichter sind immer gerne gesehen. :)`

Reply:
Wobei ich persönlich lieber weniger Smileys auf die Homepage packen würde.
@JanX2 @raphaelm opinions on that?

Reply:
`Komm einfach vorbei, der Treff ist offen für alle, neue Gesichter sind immer gerne gesehen.`

This?

Reply:
Nein, mehr Smilies. Weniger Punkte. Also, der Satz "Komm einfach […] gesehen. :)" und "Komm einfach […] gesehen :)" sind schon signifikant anders…

Reply:
Smilies auf einer Webseite? Die Startseite sollte eigentlich schlicht gehalten werden. Finde den Satz an sich schon eher unnötig. Ohne den Satz kommt man wenigstens mit den Leuten ins Gespräch.

Reply:
Ich fände den :) schon okay, aber nicht nötig.